mcodingbot/plugins/python.py

```diff
@@ -38,12 +38,17 @@ async def on_message(event: hikari.MessageCreateEvent) -> None:
         for ref in re.finditer(PEP_REGEX, event.message.content)
     ]
 
+    pep_refs = sorted(set(filter(lambda x: x < 10000, pep_refs)))
```

Comment:
```py
pep_refs = sorted({ref for ref in pep_refs if ref < 10000})
```

Reply:
that's how I originally did it but `filter` felt more clear to me

Reply:
can do set comp if u want

Reply:
I think set comp is more clear in this situation.